app/javascript/components/pages/SponsorUs.jsx

```diff
@@ -16,38 +12,28 @@ const SponsorUs = () => {
     return (
         <SharedLayout>
             <PageTitleWithContainer text="Sponsor Us" />
-            <SponsorUsWidget />
 
-            <Banner>
-                Have something else in mind? Let&apos;s talk!
-                <Button type="white" className="ml-0 md:ml-5 mt-5 md:mt-0">
+            <div className="text-xl leading-8 max-w-[500px] mx-auto mb-24 text-center">
+                <p>Thank you for your interest in sponsoring WNB.rb!</p>
+                <p className="mt-10">
+                    We are in the process of revamping our sponsorship levels and are not accepting
```

Comment:
Wonder if we can flip this terminology to focus more on the positive and mostly so that we're not explicitly saying "we're not accepting you". Subtle, but something like,

"We are in the process of revamping our sponsorship levels for all new sponsors. If you would like to be notified when these levels are ready, please fill out{' '}."

Reply:
Another thought - I wonder if there's a way we can also incorporate some sort of "In the interim, please encourage your women / non-binary engineers to [join our community](LINK)." Don't really know perfect phrasing here, but something that gets at "An integral part of all sponsorships is supporting folks from companies we sponsor" or "Getting to know engineers from your company allows our sponsorship relationship to thrive"

Reply:
I love both of these thoughts. Will incorporate and re-ping you.

Reply:
Let me know how you feel about this:

<img width="1440" alt="Screen Shot 2022-05-27 at 5 38 44 PM" src="https://user-images.githubusercontent.com/9601737/170792394-bbcebdbd-2849-4b83-94a2-a8338265ca5d.png">